examples/app/app.ts

```diff
@@ -0,0 +1,4 @@
+const el: HTMLDivElement = document.createElement('div');
+el.innerText = 'Hello, TypeScript';
```

Comment:
one larger typescript example would be a bit more realistic than this toy one

Reply:
Agreed. I moved most of the assertions from rules_typescript to `/e2e/ts_library/...`. That's a better home for most of them. The `/e2e/app` can just be a really good example without worrying about assertions.